composer.json

```diff
@@ -14,9 +14,14 @@
     "require": {
         "php": ">=5.3.2",
         "twig/twig": "1.18.*",
-        "erusev/parsedown-extra": "dev-master@dev"
+        "erusev/parsedown-extra": "dev-master@dev",
```

Comment:
@theshka (https://github.com/picocms/Pico/commit/968dc183db9fe57ad94fb5e2a536e52fc0d13243), are there any reasons that oppose using e.g. `0.7.*` instead?